src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -142,55 +141,54 @@ func (s *Service) cmdGithubCreate(roomID, userID string, args []string) (interfa
 	return gomatrix.TextMessage{"m.notice", fmt.Sprintf("Created issue: %s", *issue.HTMLURL)}, nil
 }
 
-func (s *Service) cmdGithubComment(roomID, userID string, args []string) (interface{}, error) {
+const cmdGithubReactUsage = `!github react [owner/repo]#issue (+1|-1|laugh|confused|heart|hooray)`
```

Comment:
i wasn't suggesting a literal `:+1:` to upvote - more that if the example syntax gave the emoji in terms of their emoji-one colon form, then Riot users could either type:

`!github react matrix-org/synapse#1234 :+1:`
or
`!github react matrix-org/synapse#1234 +1`
or
`!github react matrix-org/synapse#1234 👍`
and they would all work.

Perhaps we can include uncertain & grin (or their : codes?) for the other ambiguous ones?  Or at least the : codes?